connector/spanmetricsconnector/config.go

```diff
@@ -51,6 +51,12 @@ type Config struct {
 	// Optional. See defaultResourceMetricsCacheSize in connector.go for the default value.
 	ResourceMetricsCacheSize int `mapstructure:"resource_metrics_cache_size"`
 
+	// ResourceMetricsKeyAttributes filters the resource attributes used to create the resource metrics key hash.
+	// This can be used to avoid situations where resource attributes may change across service restarts, causing
+	// metric counters to break (and duplicate).
+	// e.g. ["service.name", "telemetry.sdk.language", "telemetry.sdk.name"]
```

Comment:
I like how you've given an example here that shows a potentially idiomatic configuration for `resource_metrics_key_attributes`.

What do you think about adding a link here to the opentelemetry semantic conventions web page so users can lookup what other resource keys they can add?

Reply:
Done.

Reply:
One thing that would be nice to clarify would be what happens if these are missing?